window/WindowAsAService/readme.md

```diff
@@ -0,0 +1,20 @@
+# Window As a Service
+
+This demo project shows how to use the `<TelerikWindow>` as a service in your application. You can `[Inject]` it in the desired pages and easily reuse it.
+
+## To use the `<TelerikWindow>` as a service
+
+* Create a service that creates a window instance. You can see an example of such a service fro the **Services** folder in this demo application and the `WindowService.cs` file. 
+    * Register that service as a `Singleton` to insure that the same instance of the `TelerikWindow` will be used across the entire application (`builder.Services.AddSingleton<WindowService>();`).
+* Create a custom component that builds the `<TelerikWindow>` instances. You can see an example of such a service fro the **Components** folder in this demo application and the `WindowBuilder.razor` file. 
```

Comment:
Not sure about the technical correctness of some words - 

* what is "build" an instance?
* the `Components` folder does not contain a "service" - did you mean the "Services" folder instead?

```suggestion
* Create a custom component that builds the `<TelerikWindow>` instances. You can see an example of such a service in the **Components** folder in this demo application and the `WindowBuilder.razor` file. 
```